<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task9</class>
 <widget class="QMainWindow" name="Task9">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task9</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>661</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xMinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xMaxBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>530</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yMinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yMaxBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>570</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>530</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>570</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_er" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>601</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1315</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
